webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -177,6 +181,14 @@ export default {
     updateEditorContent(value) {
       // this.form.content = value
       this.$refs.contributionForm.update('content', value)
+
+      this.disabled = true
+      this.n = value.replace(/<\/?[^>]+(>|$)/gm, '').length
+      this.form.contentLength = this.n
+
+      if (this.n > this.formSchema.content.min && this.n < this.formSchema.content.max) {
+        this.disabled = false
```

Comment:
`this.n` should be a local `let` variable and not be global to this component …

Reply:
amended